aces/Loris
# [Documentation] Delete survey_accounts.md

## Brief summary of changes
This file is a duplicate of the readme file within survey_accounts and should be deleted.

#### Testing instructions

1. Ensure the right file has been deleted
2. Ensure there is no information in this file that should be in the existing Readme file

#### Links to related tickets (GitHub, Redmine, ...)

* https://github.com/aces/Loris/issues/5196
